### PR TITLE
Add uwp

### PR DESCRIPTION
**Pull request by codergrl**
@nCastle1 please review
- added UWP project
- added script to copy resources from WPF
- wired authentication - using the Password Vault in UWP
- merged attachments work from approved `dev` branch and fixed references to get successful build

### CODE REVIEW

File: src/DataCollection.UWP/MainPage.xaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    x:Class="Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    xmlns:utils="using:Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.Utils"
+    xmlns:converters="using:Esri.ArcGISRuntime.ExampleApps.DataCollection.Shared.Converters"
+    RequestedTheme="Dark"
```
Comment (nCastle1) [inline src/DataCollection.UWP/MainPage.xaml]: I recommend removing the theme request and using the user's Theme. I tested with Light and Dark on my PC and both worked quite well - only change needed was removing explicit text color on the title.

File: src/DataCollection.UWP/App.xaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿<Application
+    x:Class="Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP.App"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DataCollection.UWP"
+    RequestedTheme="Light">
```
Comment (nCastle1) [inline src/DataCollection.UWP/App.xaml]: Explicit theme (see earlier comment)

File: src/DataCollection.UWP/MainPage.xaml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        </esri:MapView>
+
+        <!-- Compass control that displays when user rotates the map -->
+        <toolkit:Compass GeoView="{Binding ElementName=MapView, Mode=OneWay}" AutoHide="True"
+                      Margin="20"  Grid.Column="2" Grid.Row="1" />
+
+        <!-- Title bar with top buttons-->
+        <CommandBar Grid.Row="0" Grid.ColumnSpan="3">
+            <CommandBar.Content>
+                <TextBlock Text="{x:Bind MainViewModel.MapViewModel.Map.Item.Title, Mode=OneWay}"  Foreground="White" FontSize="25" FontWeight="SemiBold"
```
Comment (nCastle1) [inline src/DataCollection.UWP/MainPage.xaml]: Only other change needed for fully dynamic themes is removing the explicit foreground color. I tested and the theme picks the right color consistently.

File: src/DataCollection.Shared/Properties/Settings.cs [hunk 4]
```diff
@@ -31,11 +35,19 @@ public class Settings
         private static Settings _instance;
 
         // set the path on disk for the settings file
-        private static string _settingsPath = Path.Combine(GetFolderPath(SpecialFolder.LocalApplicationData),
+#if WPF
+        private static string _localFolder = GetFolderPath(SpecialFolder.LocalApplicationData);
+#elif NETFX_CORE
+
```
Comment (nCastle1) [inline src/DataCollection.Shared/Properties/Settings.cs]: Extra newline?

File: src/DataCollection.UWP/MainPage.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+using Windows.UI.Xaml.Controls;
+
+namespace Esri.ArcGISRuntime.ExampleApps.DataCollection.UWP
+{
+    /// <summary>
+    /// A map page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MainPage : Page
+	{
+		public MainPage()
```
Comment (nCastle1) [inline src/DataCollection.UWP/MainPage.xaml.cs]: Looks like this is using tabs instead of spaces

**Review by nCastle1 — APPROVED** [review]
Looks great! Only a few nitpicks

File: src/DataCollection.UWP/MainPage.xaml.cs (hunk 5 above)
Comment (codergrl) [reply src/DataCollection.UWP/MainPage.xaml.cs]: Microsoft did it 😆

**Pull request merged**